components/atom/input/src/Input/index.js

```diff
@@ -1,16 +1,31 @@
 import {forwardRef} from 'react'
 
+import cx from 'classnames'
 import PropTypes from 'prop-types'
 
-import {SIZES} from '../config.js'
-import Input, {inputSizes, inputStates} from './Component/index.js'
+import {
+  BASE_CLASS_AREA_FOCUSABLE,
+  BASE_CLASS_ITEM,
+  SIZES,
+  TYPES
+} from '../config.js'
+import Mask from '../Mask/index.js'
+import Password from '../Password/index.js'
+import Input from './Component/index.js'
 import InputAddons from './Wrappers/Addons/InputAddons.js'
 import InputButton from './Wrappers/Button/InputButton.js'
 import InputIcons from './Wrappers/Icons/InputIcons.js'
 
+const componentType = {
+  undefined: props => [Input, props],
```

Comment:
I am afraid, but you can define `TYPE.DEFAULT` and apply [this](https://github.com/SUI-Components/sui-components/pull/2310/files#diff-5c30bc539325f4a10f2839dc26fe851a3f1b7c046503095e45ef110814a9f882R28) as default args [see](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) 
`type = TYPE.DEFAULT`